src/Spectre.Algorithms/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
I think this file is autogenerated. Even if I place license header here, it will be lost.